src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/BalanceSheetServiceImpl.java

```diff
@@ -37,49 +39,61 @@
 
     @Autowired
     private ApiClientService apiClientService;
-    
+
     @Autowired
     private ValidationContext validationContext;
 
     @Autowired
     private CompanyService companyService;
 
+    @Autowired
+    private DebtorsService debtorsService;
+
     private AccountsDatesHelper accountsDatesHelper = new AccountsDatesHelperImpl();
 
     private static final UriTemplate SMALL_FULL_URI =
-            new UriTemplate("/transactions/{transactionId}/company-accounts/{companyAccountsId}/small-full");
+            new UriTemplate("/transactions/{transactionId}/company-accounts/{companyAccountsId" +
+                    "}/small-full");
 
-    private static final UriTemplate CURRENT_PERIOD_URI = new UriTemplate(SMALL_FULL_URI.toString() + "/current-period");
-    private static final UriTemplate PREVIOUS_PERIOD_URI = new UriTemplate(SMALL_FULL_URI.toString() + "/previous-period");
+    private static final UriTemplate CURRENT_PERIOD_URI =
+            new UriTemplate(SMALL_FULL_URI.toString() + "/current-period");
+    private static final UriTemplate PREVIOUS_PERIOD_URI =
+            new UriTemplate(SMALL_FULL_URI.toString() + "/previous-period");
 
     @Override
-    public BalanceSheet getBalanceSheet(String transactionId, String companyAccountsId, String companyNumber)
+    public BalanceSheet getBalanceSheet(String transactionId, String companyAccountsId,
+            String companyNumber)
             throws ServiceException {
 
         ApiClient apiClient = apiClientService.getApiClient();
 
-        CurrentPeriodApi currentPeriodApi = getCurrentPeriod(apiClient, CURRENT_PERIOD_URI, transactionId, companyAccountsId);
+        CurrentPeriodApi currentPeriodApi = getCurrentPeriod(apiClient, CURRENT_PERIOD_URI,
+                transactionId, companyAccountsId);
         PreviousPeriodApi previousPeriodApi = null;
 
         CompanyProfileApi companyProfileApi = getCompanyProfile(companyNumber);
 
         if (isMultipleYearFiler(companyProfileApi)) {
-            previousPeriodApi = getPreviousPeriod(apiClient, PREVIOUS_PERIOD_URI, transactionId, companyAccountsId);
+            previousPeriodApi = getPreviousPeriod(apiClient, PREVIOUS_PERIOD_URI, transactionId,
+                    companyAccountsId);
         }
 
         BalanceSheetHeadings balanceSheetHeadings = getBalanceSheetHeadings(companyProfileApi);
 
-        BalanceSheet balanceSheet = transformer.getBalanceSheet(currentPeriodApi, previousPeriodApi);
+        BalanceSheet balanceSheet = transformer.getBalanceSheet(currentPeriodApi,
+                previousPeriodApi);
 
         balanceSheet.setBalanceSheetHeadings(balanceSheetHeadings);
 
         return balanceSheet;
     }
 
-    private CurrentPeriodApi getCurrentPeriod(ApiClient apiClient, UriTemplate uri, String transactionId, String companyAccountsId) throws ServiceException {
+    private CurrentPeriodApi getCurrentPeriod(ApiClient apiClient, UriTemplate uri,
```

Comment:
We don't need to pass `UriTemplate uri` to this private method it is always the `CURRENT_PERIOD_URI` which is available to the whole class.